modules/applications-create-using-cli-template.adoc

```diff
@@ -12,6 +12,7 @@ application.
 
 Create an application from a stored template, for example:
 
+[source,terminal]
 ----
 $ oc create -f examples/sample-app/application-template-stibuild.json
 $ oc new-app ruby-helloworld-sample
```

Comment:
Perhaps we can break these commands into two separate code chunks so the copy button is relevant?

Reply:
Resolved.